documentation/modules/source-vm-prerequisites.adoc

```diff
@@ -10,7 +10,6 @@ The following prerequisites apply to all migrations:
 
 * ISO/CDROM disks must be unmounted.
 * Each NIC must contain one IPv4 and/or one IPv6 address.
-* The VM operating system must be certified and supported for use as a link:https://access.redhat.com/articles/973163#ocpvirt[guest operating system with {virt}] _and_ for link:https://access.redhat.com/articles/1351473[conversion to KVM with `virt-v2v`].
```

Comment:
the first part, ` The VM operating system must be certified and supported for use as a link:https://access.redhat.com/articles/973163#ocpvirt[guest operating system with {virt}]`, should remain here as it also applies to other providers. only the part about conversion to KVM should move to vmware-prerequisites